### ISSUE TITLE
Deleting 

### ISSUE THREAD

**Issue by kedarv** [facebook/docusaurus#2782]
## 🐛 Bug Report

Removing the following from a _clean_ install of docusaurus v2 causes 404s:
```
        blog: {
          showReadingTime: true,
          // Please change this to your repo.
          editUrl:
            'https://github.com/facebook/docusaurus/edit/master/website/blog/',
        },
```

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(Write your answer here.)

## To Reproduce

(Write your steps here:)

1. run `npx @docusaurus/init@next init my-website classic`
2. delete the entire `blog` item
3. observe navigating to `docs/doc1`  results in a 404

## Expected behavior

docs/doc1 should not 404 when in docs-only mode

## Actual Behavior
I get a 404 when accessing docs/doc1

## Your Environment
Node: v13.10.1
Chrome 81.0.4044.138

**Comment by kedarv** [comment]
actually, I'm not sure what's causing 404s. but, it seems like routing is all sorts of messed up by default. I can't tell if I'm doing something wrong, but really all I've done is cloned the project and created a page to see this 404ing.

**Comment by lex111** [comment]
This actually was fixed yesterday in #2777; strange, I assumed that @yangshun already published a bug fix release :man_shrugging:.

**Comment by lex111** [comment]
Fixed in v2.0.0-alpha.56.

**Comment by kedarv** [comment]
Thanks @lex111!
